main.tf

```diff
@@ -23,6 +23,14 @@ locals {
       subscription_id = "1c4f0704-a29e-403d-b719-b90c34ef14c9"
       name            = "hmcts-nonprod"
     }
+    qa = {
```

Comment:
hmm you've added qa but aren't using it? 

we don't use this anyway?
See the CFT definition where perftest uses nonprod
https://github.com/hmcts/cnp-aks-rbac-pipelines/blob/master/variablegroups/PERFTEST-AKS-Common.json#L31

can you remove the qa and ethosldata definitions, the mappings look correct

Reply:
ah I missed removing that after realising it wasn't used :)